tests/db-setup.test.ts

```diff
@@ -257,6 +257,15 @@ describe('Recording a ReBench execution from payload files', () => {
 
     const timeline = await db.query('SELECT * from Timeline');
     expect(timeline.rowCount).toEqual(1);
+
+    // tests to see if small payload has seperated each iteration
+    // into an array of length 1
+    const expectedValues = [[383.821], [432.783], [482.53]];
+
+    for (let index = 0; index < measurements.rows.length; index++) {
```

Comment:
Looks like you could simplify this with a `for ... of` loop over `measurements.rows`.